### ISSUE TITLE
Rephrase how powerful and complex Xnet is

### ISSUE THREAD

**Issue by ninjachicken1** [FTBTeam/FTB-Modpack-Issues#3109]
### Modpack

FTB University 1.16

### Is your suggestion related to a problem? Please describe.

The introduction chapter quest for XNet calls it "More powerful than pipes from Mekanism etc, but are also much more powerful". First of all I only just noticed a grammar error while typing that - leave out the word "are" - but I'd like to point out how misleading this is. I know my way around Mekanism (aside from the nuclear stuff) pretty well, and I've actually got some in game examples of where I tried substituting Mekanism Logistics with XNet. 

I'll start of with a case where XNet is less Complex than Mekanism. Here is my first attempt to automate the Osmium Compressor and every Metallurgic Infuser recipe using Mekanism's Logistical Sorters. Please forgive the mess, I had the idea of stuffing my entire base in compact machines and abandoned the world because things weren't working as well as I thought.
![2023-04-06_00 50 43](https://user-images.githubusercontent.com/3670136/232119010-16ebc46f-ad4c-4e8a-af92-96703cabe883.png)
As you can see, it takes up a lot of space. It took a lot of effort to colour the transporters correctly, and the Osmium Compressor and Steelmaking had to be on a completely separate line so that it the Iron and Osmium they use didn't get infused with redstone. Heres a better look at the Sorter for that line:
![2023-04-14_18 44 08](https://user-images.githubusercontent.com/3670136/232121438-b9c492ad-7eab-4a2b-a046-8a04844e26ca.png)

Now look how I did it in my new save.
![2023-04-14_19 00 56](https://user-images.githubusercontent.com/3670136/232122392-0b8ca6e2-6f4e-42ba-aab2-1c523de7a05c.png)
As you can see, XNet made everything much tidier. It still uses two input chests with their own ME Interface for each processing line, but this time they're Ender Chests. Instead of thinking about where to put Transporters and Cables, and thinking about colours and ensuring the two processing lines don't touch, all I had to do was stack 7 Advanced Connectors. I could have made the physical space even more tidy by adding only using 1 output chest and putting it on top of the other two, but I instead did it like this so the machines would just dump it in the same inventory space without much extra fiddling.
![2023-04-14_18 42 23](https://user-images.githubusercontent.com/3670136/232124819-e6335acc-8453-4557-8cc2-655c314d3cca.png)
Here's how it ticks. Each "line" uses two item channels, one for infusions, the other for the main item. If the output was going to XNet instead of directly to a chest, that would require another channel. Apart from that all I had to do is filter the items, choose the right side and set extract to stack. I could even use copy/paste to make things slightly easier.

So you can now see that it can be less complex than Mekanism. But more powerful? That's a matter of opinion. Here's what my base looked like within 24 hours of starting a new save:
![2023-03-16_11 53 47](https://user-images.githubusercontent.com/3670136/232130384-8e211a56-9e27-4305-8e56-f6a5c674d203.png)
As you can see, I've got a gas-burning generator, 5x ore, and sulfur generated from cobblegen. Between taking this screenshot and tearing it down to build the next example, I upgraded the machines to Ultimate Factories, which was overkill but there were no issues with power or chemicals not being produced or transported quick enough, even with only basic pipes/tubes/transporters/cables and the red ones I got as quest rewards. The only bottleneck was the speed of the Crystallizer. With Ender Chests, I can easily automate 5x Ore Processing and Furnace recipes, all with this setup.

At this point I should probably mention something about my playstyle. See, I came to modded Minecraft through Factorio, which was inspired by IndustrialCraft and BuildCraft. I didn't discover modded Minecraft until I'd clocked 3000 hours in Factorio and beat Seablock (which I understand is kind of like Gregtech Skyblock). Because of that I came in with the mentality that I was going to make more machines to scale up my processing, and [when I found a setup with a flowchart](https://imgur.com/gallery/Sqai8) I decided I'd try it with all logistics done by XNet, seeing as it was more "powerful". Here's how it "works".
![2023-04-14_21 27 59](https://user-images.githubusercontent.com/3670136/232148603-8fbbc8eb-a794-43bc-940f-c528dccf9158.png)
Each stage of processing (Chemical Production, Ore>Crystals, Injection, Purifying, Crushing, Enriching, Smelting) is on it's own network, all or which are connected through Routers. The idea is:

1. They all pass to the next stage
2. Everything is inserted into the machines round robin
3. AE2 can use each stage for processing
4. The Ultimate factories can process ore faster than the Crystallizers so they can catch up when it's done with non-ore processing.

What you can't see from this screenshot is that only the top row of machines is actually handling everything, and it's running worse than my starter base (apart from the fact AE2 can use each stage now). The reason for this is because, while XNet can handle the Items just fine, Energy and Fluids are another matter entirely. I first noticed this when trying to fill the Thermal Evaporation Plants (26 of them) with XNet and an Infinite Water Source. It started filling one of them extremely slowly. Then I replaced xnet with Ultimate Pipes just for the TEP's, and it was done in no time. Then I actually set up Ore Processing and I found that the more power-hungry machines kept runnning out, and XNet was only refilling one of them with power at a time, and not quickly. On it's own, that makes my setup run worse, but the kicker is that it breaks the first stage.

See, the idea was that 1 Dissolution Chamber and Washer would supply slurry to 5 Crystallizers. This was done by having Xnet only extract clean slurry when the washer has => 1000 of it, and to only insert 200 at a time, exactly enough to make 1 crystal. Problem is Power issues kept them from all running at the same time, meaning they'd often end up with less than 200 slurry in them, meaning they couldn't process it and accept the next ore unless I fixed it manually with a Gauge Dropper. (Speaking of, maybe add a quest for that?) Right now my ore processing is only using one Crystallizer, meaning it's just running worse than it did before. Besides that, there's no documentation on whether XNet logic uses OR or AND, and I can't figure out how to let it check 5 Crystallizers.

Basically XNet is more powerful than Mekanism when it comes to logic and sorting, but throughput is a very different story. And the logic isn't powerful enough to fix the issues that causes. I haven't figured out what I'n going to use to scale up Ore Processing. Maybe PNC when I get to that.
